lib/entities/parse_address_from_domain.dart

```diff
@@ -1,10 +1,13 @@
+import 'package:cake_wallet/core/yat_service.dart';
+import 'package:cake_wallet/di.dart';
 import 'package:cake_wallet/entities/openalias_record.dart';
 import 'package:cake_wallet/entities/parsed_address.dart';
 import 'package:cake_wallet/entities/unstoppable_domain_address.dart';
-import 'package:cake_wallet/store/yat/yat_store.dart';
-import 'package:cw_core/wallet_type.dart';
+import 'package:cake_wallet/entities/emoji_string_extension.dart';
 
-const unstoppableDomains = [
+class ParseAddressFromDomain {
```

Comment:
Rename class, as you parse not only from domain, i assume you can name it as "AddressResolver" and rename parse method to resolve.

Reply:
alright, done